### PR TITLE
Added healer stat, hps, HealFraction.

### PR DESCRIPTION
**Pull request by DzenDyn**
I ask to consider my suggestions.
I do not have enough in-game statistics for healing classes.
This add healstat to pasted info.

### CODE REVIEW

File: CasualMeter/res/servers.txt [hunk 1]
```diff
@@ -20,9 +20,9 @@
...
 208.67.49.60 NA Highwatch
 
-91.225.237.5 RU Arun
-91.225.237.6 RU Kaitor
-91.225.237.7 RU Arborea
-91.225.237.8 RU Lorkada
-91.225.237.9 RU Baldera
-91.225.237.12 RU Westonia
\ No newline at end of file
+91.225.237.5 RU Арун
```
Comment (DzenDyn) [inline CasualMeter/res/servers.txt]: Russian names for russian servers.

File: CasualMeter/ShellViewModel.cs [hunk 2]
```diff
@@ -288,11 +288,18 @@ private void PasteStats(PastePlayerStatsMessage obj)
...
-                if (sb.Length + playerText.Length > maxLength)
+                var playerText = first ? "" : " | ";
+
+                if (playerInfo.Player.Class != PlayerClass.Mystic && playerInfo.Player.Class != PlayerClass.Priest)
+                {
+                    playerText += placeHolder.Replace(SettingsHelper.Instance.Settings.DpsDmgPasteFormat);
+                }
+                if (playerInfo.Player.Class == PlayerClass.Mystic || playerInfo.Player.Class == PlayerClass.Priest)
+                {
+                    playerText += "HealStat: " + playerInfo.Player.Name + " HPS:" + playerInfo.Dealt.Hps + "(" + playerInfo.Dealt.Heal + ") — " + playerInfo.Dealt.HealFraction.ToString("P1").Replace(" ", string.Empty);
```
Comment (DzenDyn) [inline CasualMeter/ShellViewModel.cs]: Cant add with placeholder, therefore i added it like this.

**Comment by lunyx** [comment]
I don't mind updating the RU server names, but I am not going to add Heals per second.  It gets artificially inflated by overhealing people and is not important for determining the quality of a healer.  The main branch focuses on providing a meter primarily for use in recording damage on high score runs, and in these scenarios, mystics can even output 500k DPS.

**Comment by lunyx** [comment]
Updated with RU server names.

**Pull request closed without merging**